app/templates/index.html

```diff
@@ -118,6 +118,12 @@
                             <label for="config-new-tab">Open Links in New Tab: </label>
                             <input type="checkbox" name="new_tab" id="config-new-tab" {{ 'checked' if config.new_tab else '' }}>
                         </div>
+                        <div class="config-div config-div-view-image">
+                          <label for="config-view-image">View image: </label>
+                          <input type="checkbox" name="view_image" id="config-view-image" {{ 'checked' if config.view_image else '' }}>
+                          <div><span class="info-text"> — Adds the View Image option.
+                            The thumbnails are blurry because mobile thumbnails are smaller.</span></div> 
```

Comment:
Just some improved clarification here.

```suggestion
                          <div><span class="info-text"> — Adds the "View Image" option to image search results.
                            Note: This will cause image result thumbnails to be lower resolution.</span></div> 
```